### PR TITLE
Clean up json output

### PR DESCRIPTION
**Pull request by ethanfrey**
Adjust naming of a few fields (lower case), and pretty print the rpc output for nice viewing

### CODE REVIEW

File: rpc/lib/server/handlers.go [hunk 1]
```diff
@@ -538,7 +538,7 @@ func (wsc *wsConnection) writeRoutine() {
 				return
 			}
 		case msg := <-wsc.writeChan:
-			jsonBytes, err := json.Marshal(msg)
+			jsonBytes, err := json.MarshalIndent(msg, "", "  ")
```
Comment (melekes) [inline rpc/lib/server/handlers.go]: Shouldn't it be the responsibility of a client if it wants JSON to look pretty? `curl ... | json_pp` for example

Comment (ebuchman) [inline rpc/lib/server/handlers.go]: Yes.

File: types/validator_set.go [hunk 2]
```diff
@@ -23,8 +23,8 @@ import (
 // TODO: consider validator Accum overflow
 // TODO: move valset into an iavl tree where key is 'blockbonded|pubkey'
 type ValidatorSet struct {
-	Validators []*Validator // NOTE: persisted via reflect, must be exported.
-	Proposer   *Validator
+	Validators []*Validator `json:"validators"`
```
Comment (ebuchman) [inline types/validator_set.go]: Let's keep the comment but move it above the struct def

File: rpc/lib/server/handlers.go (hunk 1 above)
Comment (ethanfrey) [inline rpc/lib/server/handlers.go]: Can't do this with websockets.

I find it very nice to work with api's like this.  Especially when you check them with the webpage.  Just more readable.  And it is not much more overhead.  If you use gzip compression (if you care about bandwidth, we can add a middleware), then it goes down to a few % overhead).

Maybe this is just my aethetics going overboard.  But I see no reason not to make it more readable.

File: types/validator_set.go (hunk 2 above)
Comment (ethanfrey) [inline types/validator_set.go]: Corrected in 926fb83

File: rpc/lib/server/handlers.go (hunk 1 above)
Comment (ethanfrey) [inline rpc/lib/server/handlers.go]: Here's a nice article about lots on api usability, where I got this idea.  Linking to one section, but it is nice to take a look at the whole thing...

http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#pretty-print-gzip

Comment (melekes) [inline rpc/lib/server/handlers.go]: makes sense

Comment (ebuchman) [inline rpc/lib/server/handlers.go]: > Can't do this with websockets.

Why not? You dont need to see the pretty print until it hits a terminal or a webpage. Install a json pretty printer plugin. I do and its way better than just using MarshalIndent (colors and close/open functionality and so on)

**Pull request merged**